src/kube-runtime/src/init/sshd.sh

```diff
@@ -20,22 +20,69 @@
 SSHD_BIN=/usr/sbin/sshd
 CONFIG_DIR=/usr/local/pai/sshd
 
-if [ -f /usr/sbin/sshd ] ; then
-    if [ -z "$PAI_SSH_PUB_KEY" -o -z "$PAI_CONTAINER_SSH_PORT" ] ; then
-        echo "no private key or port provided" >&2
-    else
-        mkdir -p $CONFIG_DIR
+function prepare_ssh()
+{
+  mkdir /root/.ssh
+  touch /root/.ssh/authorized_keys
+  chmod 600 /root/.ssh/authorized_keys
+
+  mkdir -p /var/run/sshd
```

Comment:
do not need?

Reply:
Better have this in case the files are not created.